.env.example

```diff
@@ -13,6 +13,10 @@ NEXT_PUBLIC_TORUS_RPC_URL="wss://api.testnet.torus.network"
 # The cache provider URL used to get some data from the cache.
 NEXT_PUBLIC_TORUS_CACHE_URL="https://dev.cache.torus.network/"
 
+# The environment setting for the Torus network: 
+# Use empty string "" for mainnet, or 'testnet' for testnet environment
+NEXT_PUBLIC_CHAIN_ENV=""
```

Comment:
I don't think we should default to `mainnet`, as it is a sensitive environment. We don't want code that should run in the development environment to accidentally use `mainnet`. Therefore, it should either default to `testnet` or something else, or the variable should be strictly required.